cmd/app/main.go

```diff
@@ -74,7 +76,7 @@ func main() {
 	addSat := func(r, orb, hper float64, s string) {
 		if stor != nil {
 			sat := JSat{R: r, Orb: orb, HPer: hper, Text: s}
-			id := strconv.Itoa(stor.Length() + 1)
+			id := storPrefix + strconv.Itoa(stor.Length()+1)
```

Comment:
Seems like `go fmt` is needed.

Reply:
The code seems ok to me, even in play.golang.org as in [https://play.golang.org/p/kKQMfw6hfAi](https://play.golang.org/p/kKQMfw6hfAi).
Can you point me exactly at what was wrong, please ?

Reply:
Just noticed the spaces ware removed around `+`. Not critical though.